content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -154,9 +154,11 @@ without any problems, the kubelet resets the restart backoff timer for that cont
 
 A Pod has a PodStatus, which has an array of
 [PodConditions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podcondition-v1-core)
-through which the Pod has or has not passed:
+through which the Pod has or has not passed. Kubelet manages the following
+PodConditions:
 
 * `PodScheduled`: the Pod has been scheduled to a node.
+* `PodHasNetwork`: the Pod sandbox has been successfully created and networking configured.
```

Comment:
Is there a way to indicate within the bullet list that `PodHasNetwork` is in Alpha? We mention this in the dedicated section but when I tried to inline the `{{< feature-state for_k8s_version="v1.25" state="alpha" >}}` bits in the bullet list, the list looked quite messed up and hard to read.

Reply:
```suggestion
* `PodHasNetwork`: (alpha feature; must be [enabled explicitly](#pod-has-network)) the
  Pod sandbox has been successfully created and networking configured.
```